src/components/EditCard/PhysicalAttr.vue

```diff
@@ -49,35 +48,57 @@
       </div>
 
       <div v-if="fclass === 'Bed'">
-        <div>Bed Features</div>
-        <v-checkbox v-model="hasFrame" label="Has bed frame?" />
-        <v-checkbox v-model="hasBoxSpring" label="Has box spring?" />
+        <h3>Bed Features</h3>
+        <v-checkbox
+          v-model="hasFrame"
+          label="Has bed frame?"
+          hide-details
+          :readonly="readonly"
+        />
+        <v-checkbox
+          v-model="hasBoxSpring"
+          label="Has box spring?"
+          hide-details
+          :readonly="readonly"
+        />
       </div>
 
-      <div>Other physical attributes</div>
-      <v-checkbox v-model="heavy" label="Heavy furniture" />
+      <h3 class="pt-4">Other physical attributes</h3>
+      <v-checkbox
+        class="pb-3"
+        v-model="heavy"
+        label="Heavy furniture"
+        hide-details
+        :readonly="readonly"
+      />
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
 import { Prop, Component } from "vue-property-decorator";
-import { Size } from "@/data/Furniture";
+import { Size, Furniture } from "@/data/Furniture";
 import { FClass, Material } from "@/data/furniture/Physical";
 
 @Component
 export default class PhysicalAttr extends Vue {
   @Prop()
-  fclass!: FClass;
+  readonly furniture!: Furniture;
+
+  @Prop({ default: false })
+  readonly readonly!: boolean;
+
+  @Prop()
+  readonly fclass!: FClass;
 
   materials = Object.keys(Material);
 
   size = -1;
 
   material = "";
 
-  showAltMaterial = false;
+  showAltMaterial = !!this.readonly;
```

Comment:
Just curious what !! is used for here

Reply:
Oooo, good question, I actually just learned about this as I was writing this. This is a good [blog post explaining the operator](https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054).

After reading it, I'm realizing that its utility mostly comes from non-boolean values. So, I'm going to make a push that fixes this before merging.